spec/classes/document_search_spec.rb

```diff
@@ -209,6 +209,24 @@
       end
     end
 
+
+    %w[doc docx pdf ppt xls xlsx].each do |ext|
+      context 'when the results contain demoted and non-demoted file types' do
+        let(:query) { 'rutabaga' }
+        before do
+          Document.create(common_params.merge( title: 'other', description: 'Rutabagas', content: 'other', path: "http://www.agency.gov/dir1/page1.#{ext}"))
```

Comment:
Using the default `let` variables will clean this up, but in general try to keep your code lines under 80 characters. That helps anyone looking at your code to be able to look at code files side by side without having to scroll left & right.